spec/factories/users.rb

```diff
@@ -0,0 +1,85 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :user_jane_doe, class: User do
```

Comment:
rename is as user. Use ffaker gem to generate Random name and other attributes
https://github.com/ffaker/ffaker

Reply:
@kth-wtag done, apu.